src/main/webapp/resources/js/pages/projects/remote/ProjectSynchronizationSettings.jsx

```diff
@@ -46,21 +46,39 @@ export function ProjectSynchronizationSettings() {
             desc: (
               <div>
                 <div>{formatDate({ date: remoteProjectData.lastUpdate })}</div>
-                <Button
-                  type="primary"
-                  onClick={() => updateSyncSettings({ forceSync: true })}
-                  disabled={
-                    (disableSyncNow ? true : false) ||
-                    (syncNowEnabledStates.includes(
-                      remoteProjectData.remoteStatus.syncStatus
-                    )
-                      ? false
-                      : true)
-                  }
-                  className="t-sync-now-btn"
-                >
-                  {i18n("ProjectRemoteSettings.syncNow")}
-                </Button>
+                <Space>
+                  <Button
+                    type="primary"
+                    onClick={() => updateSyncSettings({ markSync: true })}
+                    disabled={
+                      (disableSyncNow ? true : false) ||
+                      (syncNowEnabledStates.includes(
+                        remoteProjectData.remoteStatus.syncStatus
+                      )
+                        ? false
+                        : true)
+                    }
+                    className="t-sync-now-btn"
+                  >
+                    {i18n("ProjectRemoteSettings.syncNow")}
+                  </Button>
+
+                  <Button
+                    type="primary"
+                    onClick={() => updateSyncSettings({ forceSync: true })}
+                    disabled={
+                      (disableSyncNow ? true : false) ||
+                      (syncNowEnabledStates.includes(
+                        remoteProjectData.remoteStatus.syncStatus
+                      )
+                        ? false
+                        : true)
+                    }
+                    className="t-sync-now-btn"
```

Comment:
Change this className to `t-sync-force-btn`

Reply:
Done in ee5e5c5